test/test_binance_market.py

```diff
@@ -316,12 +316,37 @@ def test_cancel_all(self):
         quantize_ask_price: Decimal = self.market.quantize_order_price(symbol, ask_price * 1.5)
 
         self.market.buy(symbol, quantized_amount, OrderType.LIMIT, quantize_bid_price)
+        self.market.buy("LOOMETH", amount)
         self.market.sell(symbol, quantized_amount, OrderType.LIMIT, quantize_ask_price)
 
         self.run_parallel(asyncio.sleep(1))
         [cancellation_results] = self.run_parallel(self.market.cancel_all(5))
         for cr in cancellation_results:
             self.assertEqual(cr.success, True)
+        self.market.sell("LOOMETH", amount)
+
+    def test_order_price_precision(self):
+        symbol = "IOSTETH"
+        bid_price: float = self.market.get_price(symbol, True)
+        ask_price: float = self.market.get_price(symbol, False)
+        amount: float = 0.02 / bid_price
+        quantized_amount: Decimal = self.market.quantize_order_amount(symbol, amount)
+        self.market.buy("IOSTETH", amount)
```

Comment:
Market order functions are asynchronous... it's generally not true that you'll have the IOST tokens in your wallet right after calling this function.

The way we do these market tests is to assume the wallet have the tokens needed to carry out the tests to begin with. That's why, e.g. the `test_buy_and_sell()` function starts by asserting you have a certain amount of ETH to start with. I'd advise you to do a similar assert in the beginning to make sure the account has enough ETH and IOST to make the limit orders first, and make the limit orders far, far away from the trading price s.t. they won't be traded.